src/crypto.ts

```diff
@@ -9,6 +9,9 @@ import {
 import { getPublicKey } from './util/publicKey'
 import { determineIsFormFields } from './util/validate'
 
+// Validation version used in `isValid` function.
+const INTERNAL_VALIDATION_VERSION = -1
```

Comment:
Nitpick but it seems a bit strange to use `-1` as a version number. Perhaps it can simply be inlined within `isValid` function if it's not being used anywhere else.

Reply:
fixed